### PR TITLE
Maps with avatars

### PR DESCRIPTION
**Pull request by ThPadelis**
Added map with the users' avatars, based on the current user's location

- Added gesture handling for the map
- Added country codes in order to search and match the user's country
- Made some minor CSS changes

### CODE REVIEW

File: src/config/config.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import keys from './keys';
 
 export default {
-	api: process.env.VUE_APP_API_URL || 'http://localhost:3000',
+	api:
+		process.env.VUE_APP_API_URL || 'https://frontend-social-api.herokuapp.com',
```
Comment (souvikbasu) [inline src/config/config.js:5]: can you please revert this change as it is needed for running the app code on localhost?

File: src/config/keys.js [hunk 2]
```diff
@@ -1,25 +1,25 @@
 export default {
```
Comment (souvikbasu) [inline src/config/keys.js:1]: please discard changes to the keys.js file

**Review by souvikbasu — CHANGES_REQUESTED** [review]

File: src/config/config.js (hunk 1 above)
Comment (ThPadelis) [reply src/config/config.js:5]: Reverted changes

File: src/config/keys.js (hunk 2 above)
Comment (ThPadelis) [reply src/config/keys.js:1]: Discarded changes
